projects/experimental/directives/block-details/block-details.style.less

```diff
@@ -0,0 +1,73 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+[tuiBlockDetails] {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font: var(--tui-font-heading-3);
+    width: 100%;
+
+    [tuiTitle] {
+        text-align: center;
+        font: var(--tui-font-text-l);
+    }
+
+    [tuiSubtitle],
+    [tuiAmountDescription] {
+        font: var(--tui-font-text-m);
+        color: var(--tui-text-02);
+
+        :host-context(tui-root._mobile) & {
+            font: var(--tui-font-text-s);
+        }
+    }
+
+    [tuiAmountDescription] {
+        margin-top: 0.25rem;
+    }
+
+    [tuiComment] {
```

Comment:
Do we maybe need this as a separate component? There's probably a good chance it will be used somewhere else